scipy/stats/_multivariate.py

```diff
@@ -1578,6 +1590,128 @@ def rvs(self, size=1, random_state=None):
+    var(alpha)
+        The variance of the multivariate beta distribution
+    entropy(alpha)
+        Compute the differential entropy of the multivariate beta distribution.
+
+    Parameters
+    ----------
+    %(_dirichlet_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    See Also
+    --------
+    dirichlet
+
+    Notes
+    -----
+    Each :math:`\alpha` entry must be positive. The distribution has only
+    support on the simplex defined by
+
+    .. math::
+        \sum_{i=1}^{K} x_i = 1
+
+    where :math:`0 < x_i < 1`.
+
+    If the quantiles don't lie within the simplex, a ValueError is raised.
+
+    The probability density function for `multivariate_beta` is
+
+    .. math::
+
+        f(x) = \frac{1}{\mathrm{B}(\boldsymbol\alpha)}
+               \prod_{i=1}^K x_i^{\alpha_i - 1}
+
+    where
+
+    .. math::
+
+        \mathrm{B}(\boldsymbol\alpha) = \frac{\prod_{i=1}^K \Gamma(\alpha_i)}
+                                     {\Gamma\bigl(\sum_{i=1}^K \alpha_i\bigr)}
+
+    and :math:`\boldsymbol\alpha=(\alpha_1,\ldots,\alpha_K)`, the
+    concentration parameters and :math:`K` is the dimension of the space
+    where :math:`x` takes values.
+
+    Examples
+    --------
+    >>> from scipy.stats import multivariate_beta
+
+    Generate a multivariate_beta random variable
+
+    >>> quantiles = np.array([0.2, 0.2, 0.6])  # specify quantiles
+    >>> alpha = np.array([0.4, 5, 15])  # specify concentration parameters
+    >>> multivariate_beta.pdf(quantiles, alpha)
+    0.2843831684937255
+
+    The same PDF but following a log scale
+
+    >>> multivariate_beta.logpdf(quantiles, alpha)
+    -1.2574327653159187
+
+    Once we specify the multivariate beta distribution
+    we can then calculate quantities of interest
+
+    >>> multivariate_beta.mean(alpha)  # get the mean of the distribution
+    array([0.01960784, 0.24509804, 0.73529412])
+    >>> multivariate_beta.var(alpha) # get variance
+    array([0.00089829, 0.00864603, 0.00909517])
+    >>> multivariate_beta.entropy(alpha)  # calculate the differential entropy
+    -4.3280162474082715
+
+    We can also return random samples from the distribution
+
+    >>> multivariate_beta.rvs(alpha, size=1, random_state=1)
+    array([[0.00766178, 0.24670518, 0.74563305]])
+    >>> multivariate_beta.rvs(alpha, size=2, random_state=2)
+    array([[0.01639427, 0.1292273 , 0.85437844],
+           [0.00156917, 0.19033695, 0.80809388]])
+
+    Alternatively, the object may be called (as a function) to fix
+    concentration parameters, returning a "frozen" multivariate beta
+    random variable:
+
+    >>> rv = multivariate_beta(alpha)
+    >>> # Frozen object with the same methods but holding the given
+    >>> # concentration parameters fixed.
+
+    """
+    def __call__(self, alpha, seed=None):
+        return multivariate_beta_frozen(alpha, seed=seed)
+
+    def _check_input(self, alpha, x):
+        x = np.moveaxis(x, -1, 0)
```

Comment:
Probably could have just done transpose. I didn't realize that `dirichlet` was only written for 2D `x`.